support/tech-resources/using-proxy-https-tracking-domain.md

```diff
@@ -23,9 +23,13 @@ This post covers how to configure a SparkPost tracking domain, provision an SSL
 
 This example uses a t2.micro Ubuntu instance on Amazon Web Services and a tracking domain of click.nddurant.com that is CNAME'd to the instance public IPv4 DNS.
```

Comment:
is this still true in your example for apache?

Reply:
I worked on my own domain, but left your example in the doc.